api/common/request_id_util.go

```diff
@@ -0,0 +1,33 @@
+package common
+
+import (
+	"context"
+
+	"github.com/fnproject/fn/api/id"
+)
+
+const (
+	maxLength = 32
```

Comment:
would possibly remove this since it's likely only bound to make us slip up and find a conflicting request id early on (this is never fun to debug, have lived that one). the basis would be that since we're creating the request ids, we shouldn't need to verify their length when we retrieve them - the counter point being more nefarious people could inject really long request ids if they want, but the boundary between lb and server is relatively secure in this respect (using https) and the chance of mishap likely exceeds the chance of users attempting to inject bad request ids. 

a possibly less error prone magic number may be something more along the lines of `1024`, which still limits the memory usage, but it's not likely we'll ever generate an id that is 1024 byte (32 on the other hand is much more likely to be exceeded).

Reply:
this is effectively unused since `id.New().String()` is a static length - we can remove this.

Reply:
I'll remove it.